src/intl/en.json

```diff
@@ -561,6 +565,59 @@
     "voting_modal_survey": {
       "title": "We'd appreciate some extra feedback",
       "selected_choice": "You're about to vote "
+    },
+    "identity_connect": {
+      "description": "**New Feature: Linked Profiles**\n\nEnhance your Profile by linking different third-party accounts used in the Decentraland DAO ecosystem.\n\nUsers with linked accounts will get an eye-catching indicator next to their username.",
+      "connect_button": "Link Accounts",
+      "cancel_button": "Maybe later"
+    },
+    "identity_setup": {
+      "title": "Choose accounts to link",
+      "timer": "Time sensitive task. {time} left to complete",
+      "timer_expired": "Signature expired. You'll have to start over.",
+      "forum": {
+        "card_title": "Forum Account",
+        "card_description": "Make comments left on proposals link back to, and be displayed on your Governance profile.",
+        "title": "Link Forum Account",
+        "title_step_1": "1. Sign message",
+        "description_step_1": "Use your web3 wallet to securely sign a message to start the linking process.",
+        "action_step_1": "Sign",
+        "helper_step_1": "First, sign the message.",
+        "title_step_2": "2. Copy to clipboard",
+        "description_step_2": "You'll be using the same content as the signature for the next step.",
+        "action_step_2": "Copy",
+        "helper_step_2": "Then, copy the signed message.",
+        "title_step_3": "3. Post on Forum thread",
+        "description_step_3": "Paste the content onto the comment box and publish to complete the linking.",
+        "action_step_3": "Open",
+        "helper_step_3": "Finally, comment the signed message on the forum thread.",
+        "card_helper": {
+          "step_1_start": "Will open up a modal",
+          "step_1_active": "Awaiting signature…",
+          "step_1_success": "Signed correctly",
+          "step_1_error": "Something went wrong, try again",
+          "step_2_start": "Signature required",
+          "step_2_active": "Click to copy to clipboard",
+          "step_2_success": "Content copied"
+        },
+        "action": "Confirm link",
+        "helper_loading": "Linking in progress...",
+        "success_text": "**Governance and Forum accounts linked successfully** 🎉\n\nNow your comments left on Proposals' forum threads will link back to, and be displayed on your Governance profile.",
+        "success_subtext": "Plus, you'll get a fancy icon next to your **username**",
+        "success_button": "Got it",
+        "error_text": "**Governance and Forum accounts could not be linked**\n\nSomething happened. It must have been aliens, or a malevolent AI but we can't be sure.",
+        "error_subtext": "Maybe just have a go at it one more time?",
+        "error_button": "Retry"
+      },
+      "discord": {
+        "card_title": "Discord",
+        "card_description": "Elevate your credibility by letting others know which wallet is behind your chat interactions."
```

Comment:
Maybe `address` instead of wallet?

Reply:
We also use the `name` when there is one available.

Reply:
How about `account`? (ethereum addresses are also known as externally owned accounts, so it works on both levels)